source-map-support.js

```diff
@@ -84,7 +84,7 @@ function retrieveSourceMapURL(source) {
 
   // Get the URL of the source map
   fileData = retrieveFile(source);
-  var re = /\/\/[#@]\s*sourceMappingURL=([^'"]+)\s*$/mg;
+  var re = /(?:\/\/[@#][ \t]+sourceMappingURL=([^\s'"]+?)[ \t]*$)|(?:\/\*[@#][ \t]+sourceMappingURL=([^\*]+?)[ \t]*(?:\*\/){1}[ \t]*$)/mg;
```

Comment:
I am reluctant to had such a complex and not commented regexp…

From which project does it come from?